### PR TITLE
Ks/readme

### PR DESCRIPTION
**Pull request by ks10825n**
Hi guys, first draft of readme, let me know any changes. You guys can pull down this PR with and see the preview of the markdown on VSC.
git fetch origin pull/`PR#`/head:`new branch name`
git checkout `new branch name`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,60 @@
...
+## Table of Contents
+
+1. [General Info](#general-info)
+2. [Demo](#demo)
+3. [Technologies Used](#technologies-used)
+4. [Contributors](#contributors)
+
+## General Info
+
+Soapstone Group is a group of like-minded individuals that came together deliver based off of client needs. For this particular product, we are creating a modernized client-facing retail web-portal, called Project Catwalk. This retail portal will provide the following function upon it's initial release and deployment:
```
Comment (kyelindholm) [inline README.md:14]:
```suggestion
Soapstone Group is a group of like-minded individuals that came together to deliver based off of client needs. For this particular product, we are creating a modern client-facing retail web-portal, called Project Catwalk. This retail portal will provide the following function upon its initial release and deployment:
```

File: README.md [hunk 2]
```diff
@@ -1 +1,60 @@
...
+### Questions & Answers
+![Questions and Answers](./demo/questionsDemo.gif)
+
+### Ratings & Reviews
+![Ratings and Reviews](./demo/ratingsDemo.gif)
+
+### Changing Product with Search Bar
+![Search Bar](./demo/changeProductDemo.gif)
+## Technologies Used
+Please note that the below are a list of all major dependencies used in the project. There could be various adapters and minor dependencies needed. Please ensure to install the necesary modules with NPM.
```
Comment (kyelindholm) [inline README.md:46]:
```suggestion
Please note that below is a list of all major dependencies used in the project. There could be various adapters and minor dependencies needed. Please ensure to install the necesary modules with NPM.
```

**Review by kyelindholm — COMMENTED** [review]
Just a couple of typos, but looks great overall! Thanks, Kevin!

File: README.md (hunk 1 above)
Comment (ks10825n) [reply README.md]: fixed!

File: README.md (hunk 2 above)
Comment (ks10825n) [reply README.md]: fixed!

**Review by kyelindholm — APPROVED** [review]

**Pull request merged**